lib/resources/order-recipients.js

```diff
@@ -0,0 +1,149 @@
+  debugLogs.push({ function: 'createOrder', params });
+  try {
+    const response = await axios.post(`${config.get('piiServerUrl')}/api/v4/programs/${contextId}/orders`, params);
+    callback(null, response.data, response);
+  } catch (error) {
+    callback({ error, debugLogs });
+  }
+};
+
+/**
+ * Higher order function for creating a `create` order function in context.
+ *
+ * NOTE: This is a v5 API endpoint, therefore ignores the `apiVersion` config prop
+ *
+ * @param {module:resources/orders~OrderContext} orderContext Order context object
+ *
+ * @returns {module:resources/orders~CreateFunc} `create` order function in context
+ *
+ * @protected
+ */
+const storeOrderRecipient = ({ programCode }) => async member => {
+  await storeOrderRecipientSchema.validate(member).then(async () => {
+    try {
+      await setV5Token();
+    } catch (error) {
+      debugLogs.push({ function: 'storeOrderRecipient.getV5Token', error });
+      // eslint-disable-next-line no-throw-literal
+      throw { errors: { error, debugLogs } };
+    }
+
+    try {
+      const {
+        data: { result },
+      } = await axios.post(`${config.get('piiServerUrl')}/api/v5/programs/${programCode}/order_recipients`, member);
+      return result;
+    } catch (error) {
+      debugLogs.push({
+        function: 'storeOrderRecipient.createOrderRecipients',
+        error: get(error, 'response.data'),
+      });
+      // eslint-disable-next-line no-throw-literal
+      throw { errors: { error, debugLogs } };
+    }
+  });
+};
+
+const getOrderRecipient = ({ programCode }) => async (orderRecipientCode, callback) => {
+  try {
+    await setV5Token();
+    const { data } = await axios.get(
+      `${config.get('piiServerUrl')}/api/v5/programs/${programCode}/order_recipients/${orderRecipientCode}`
+    );
+    callback(null, data.result, data);
+  } catch (error) {
+    callback(error);
+  }
+};
+
+const orderRecipientFactory = (contextTypeName, contextId, programCode) => {
+  if (contextTypeName !== 'programs') {
+    throw new Error('Can only create an order recipient object for programs');
+  }
+  if (!config.get('piiServerUrl')) {
+    throw new Error('piiServerUrl is not configured');
+  }
+
+  const orderContext = {
+    contextTypeName,
+    contextId,
+    programCode,
+  };
+
+  return {
+    getOrderRecipient: getOrderRecipient(orderContext),
+    create: async ({ member, ...rest }, callback) => {
+      try {
+        const orderRecipientData = await storeOrderRecipient(orderContext)(member);
+        await createOrder(orderContext)({ ...rest, ...orderRecipientData }, callback);
+      } catch (error) {
+        callback(error);
+      }
+    },
+  };
+};
+
+module.exports = orderRecipientFactory;
+
+/**
+ * Default SDK configuration options type
+ *
+ * @typedef module:resources/orders-recipients~PiiRequestBody
+ *
+ * @property {string} id Member ID
+ * @property {string} [accept_language] The member's preferred locale, in the format of Accept-Language
```

Comment:
Is this correct, i.e. that `accept_language` is optional (notice the `[` `]`s)? If so, I think we can sidestep the entire `supportedLocales` config prop. Instead, we can assume that the call will use the program's default locale…

Reply:
Just tested with Postman, `accept_language` is required, will update docs